packages/qwik-city/src/middleware/request-handler/request-event.ts

```diff
@@ -102,8 +102,15 @@ export function createRequestEvent(
     } else {
       status = statusOrResponse.status;
       statusOrResponse.headers.forEach((value, key) => {
+        if (key.toLowerCase() === 'set-cookie') {
+          return;
+        }
         headers.append(key, value);
       });
+      statusOrResponse.headers.getSetCookie().forEach((ck) => {
+        const [key, value] = ck.split(/=(.+)/);
```

Comment:
Looks great but can you split using indexOf?

Reply:
Can you re-check pls?